config/endpoints/api_particulier/cnaf_msa_quotient_familial_v2.yml

```
@@ -153,12 +157,30 @@
         > C'est le cas d'un enfant ayant quitté le foyer de ses parents et ayant ouvert son propre dossier d'allocataire pour bénéficier d'une aide (par exemple les APL ou le RSA), mais qui souhaite tout de même être rattaché à la déclaration fiscale de ses parents.
         > _Cet enfant appartiendra au même foyer fiscal que ses parents au sens de l'administration fiscale, mais à une famille différente selon la CAF ou la MSA._
 
+    - q: Dans quels cas les deux parents sont-ils associés à un même compte ?
+      a: |+
+        Les deux parents sont associés à un même compte **à partir de la déclaration du dit changement par l'allocataire, sur le site caf.fr ou sur l’application “Caf -Mon Compte”**. Ce changement a lieu lorsque les parents sont mariés, pacés ou concubins. Cette déclaration induit un même domicile, exception faite du cas d'un conjoint vivant à l'étranger dont la vie maritale est tout de même enregistrée auprès de la CAF ou la MSA.
+        
+    - q: Gestion des familles séparées
+      a: |+
+        La 
+
+    - q: Les enfants majeurs sont-ils remontés par l’API ? ?
+      a: |+
+        Oui, si l'enfant est toujours à la charge des parents. Cette charge est modulée en fonction de la rémunération de l'enfant, le cas échéant. 
+
+    - q: Dans le cas d’une famille recomposée, les enfants des deux parents sont-ils remontés sur un même compte ? 
+      a: |+
+        L'enfant sera uniquement rattaché au parent qui en a la charge. Dans le cas d'une situation de résidence alternée qui induit un partage de la charge, l'enfant apparaitra sur les deux comptes uniquement si la "résidence alternée avec partage des Allocations" est déclarée à la Caisse d'allocation. 
+
+    - q: Les enfants décédés sont-ils remontés par l’API ? ?
+      a: |+
+        Les enfants décédés ne sont plus considérés à charge le mois suivant le décès et ne sont donc plus remontés par l'API.
+
+    - q: Les parents peuvent ils être rattachés à deux régimes différents ? Par exemple, un parent rattaché à la CAF et l'autre à la MSA. 
+      a: |+
+        Le rattachement des parents ne se fait que sur un seul régime. Selon la situation de l'allocataire, un organisme sera priorisé par rapport à l'autre. 
```

```suggestion
        Les parents qui sont associés à un même compte, sont gérés par une seule caisse, la CAF ou la MSA. Selon la situation de l'allocataire, un organisme sera priorisé par rapport à l'autre. 
```

_Selon la situation de l'allocataire, un organisme sera priorisé par rapport à l'autre._ 
=> Est-ce qu'on en sait plus là-dessus ? C'est assez flou.
=> Est-ce que si, pour 2 parents (A & B) rattachés à un même compte, si je fais un appel avec l'identité du parent A, est-ce que j'aurai forcément la même caisse que si je fais l'appel avec l'identité du parent B ?

---

Par ailleurs, j'aurai remonté cette question en dessous de la question `"Dans quels cas les deux parents sont-ils associés à un même compte ?"`

---

L'appel ne peut être fait que sur l'allocataire principal donc à partir du moment où ils passent sur une seule caisse, l'appel ne fonctionnera qu'avec l'identité de l'allocataire qui est sur cette caisse 
pour la règle de priorisation, je pose la question à la CNAF